modules/claims_api/lib/bgs_service/manage_representative_service.rb

```diff
@@ -1,18 +1,53 @@
 # frozen_string_literal: true
 
-require_relative 'manage_representative_service/update_poa_request'
-
 module ClaimsApi
   class ManageRepresentativeService < ClaimsApi::LocalBGS
-    private
-
-    def make_request(**args)
-      super(
-        endpoint: 'VDC/ManageRepresentativeService',
-        namespaces: { 'data' => '/data' },
-        transform_response: false,
-        **args
-      )
+    def bean_name
+      'VDC/ManageRepresentativeService'
+    end
+
+    def read_poa_request(poa_codes: [])
```

Comment:
cf. https://github.com/department-of-veterans-affairs/bgs-catalog/blob/4234350795ba516b5491d95bfd6582264fea782c/VDC/ManageRepresentativeService/ManageRepresentativePortBinding/readPOARequest/request.xml